src/js/game/GameController.js

```diff
@@ -1301,6 +1301,42 @@ class GameController {
     });
   }
 
+  placeBlockDirection(commandQueueItem, blockType, direction) {
```

Comment:
I think `placeBlockForward` is now just a special case of this method?  Can `placeBlockForward` call `placeBlockDirection` with the third param always the forward direction?

```js
placeBlockForward(commandQueueItem, blockType) {
  const entity = this.getEntity(commandQueueItem.target);
  const forwardPosition = this.levelModel.getMoveForwardPosition(entity);
  this.placeBlockDirection(commandQueueItem, blockType, forwardPosition);
}
```

Reply:
Yeah, that's totally fine. I wasn't messing with placeBlockForward simply because I didn't know what assumptions were going to be made with previous years and what not, but that's actually a great suggestion.

Will implement. 👍

Reply:
@joshlory 
Also, placeBlockDirection() doesn't take a position for it's third arg, it takes a relative direction. So this is sufficient: 😂 
>>>  placeBlockForward(commandQueueItem, blockType) {
    this.placeBlockDirection(commandQueueItem, blockType, 0);
  }